packages/vite/src/node/config.ts

```diff
@@ -402,69 +413,88 @@ export async function resolveConfig(
   }
 
   // TODO Deprecation warnings - remove when out of beta
-  if (config.build?.base) {
+
+  const logDeprecationWarning = (
+    deprecatedOption: string,
+    hint: string,
+    error?: Error
+  ) => {
     logger.warn(
-      chalk.yellow.bold(
-        `(!) "build.base" config option is deprecated. ` +
-          `"base" is now a root-level config option.`
+      chalk.bold.yellow(
```

Comment:
**nitpick:** Let's switch this back to `chalk.yellow.bold` so the diff gets smaller